docs/product/issues/custom-views/index.mdx

```diff
@@ -0,0 +1,61 @@
+
+Custom issue views let you customize what shows up in your issue stream so you can quickly see what's most important to you. You have full control over which search query your issue stream renders by default, and which other queries are within a click's reach.
+
+
+## Creating Your First View
+
+To create your first custom issue view, navigate to the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page and click the **Add View** button. You'll see the following page:
+
+{/* TODO(msun) Insert image here once it's been updated for EA (current state of custom views is LA-specific) */}
+
+Here, you'll be able to select a view from one of the recommended options or create your own by following the instructions below:
+
+
+1. **Name your view**: Give your view a name that describes the types of issues it will show.
+2. **Set your view's search query**: Choose the search query that will be applied by default when you select this view. Don't forget to hit enter to apply the query!
+3. **Set your view's sort**: Select how you want the issues to be sorted in this view (for example, by date, events, and so on).
+4. **Save your view**: Click on the ellipsis menu within the tab and select **Save Changes**.
+
+<Note>
+If you had a default custom saved search, it'll be converted to a custom view tab and will appear as your first view automatically. Any other saved searches you created will be temporarily available in the **Add View** menu. Please save them as view tabs if you'd like to keep them.
+</Note>
+
+## Customizing Your Views
+
+
+### Updating View Queries and Sorting
+
+Views are defined by their search query and sort order. If you want to update either the query or the sort, you'll see an **unsaved changes indicator** (shown below). To save your changes, click the ellipsis menu and select **Save Changes**. To discard them, click **Discard Changes** within the ellipsis menu, or close your Sentry tab without saving.
+{/* TODO(msun) figure out how to resize this */}
+![Unsaved Changes Indicator](./img/custom_views_unsaved_changes_indicator.png)
+
+
+### Reordering Views and Setting a Default View
+
+Your can reorder your view tabs by dragging them to where you want them to be. The first tab will always be your default view and will be what you see when you first open the **Issues** page.
+
+
+### Renaming Views
+
+To rename a view, select it from the tab list. Then, either double click on the view's name or click the ellipsis menu and select **Rename**, type in a new name, and hit enter.
+
+
+## Sharing Views
+
+We built views in a way that makes them easy to share with people in your organization. To share, just copy the link to your issue view and send it to a fellow org member. The link will take them to a **temporary tab** that contains the same query and sort as your view. They'll then be able to save the view as their own by clicking **Save View**, or discard it by clicking **Discard**.
+
```

Comment:
Reminder to self: add a note that temporary tabs are not persisted, unlike normal tabs.